themes/default/content/blog/aws-lambda-functions-powered-by-graviton2/index.md

```diff
@@ -0,0 +1,439 @@
+---
+title: "AWS Lambda Functions Powered by AWS Graviton2 Processors"
+date: 2021-09-29
+meta_desc: Deploy AWS Lambda Functions powered by AWS Graviton2 Processors
+meta_image: meta.png
+authors:
+    - paul-stack
+tags:
+    - aws
+    - lambda
+    - arm64
+
+---
+
+In late 2018, [AWS launched their first EC2 instances](https://aws.amazon.com/blogs/aws/new-ec2-instances-a1-powered-by-arm-based-aws-graviton-processors/)
+powered by Arm-based AWS Graviton Processors. These instances had been optimized for performance and cost. Since that initial launch,
```

Comment:
```suggestion
powered by ARM-based AWS Graviton Processors. These instances had been optimized for performance and cost. Since that initial launch,
```

Reply:
Arm has dropped upper-case branding: https://www.arm.com/company/policies/trademarks/guidelines-brand